source/classes.tex

```diff
@@ -2260,10 +2260,9 @@
 \pnum
 \indextext{destructor!virtual}%
 \indextext{destructor!pure virtual}%
-A prospective destructor can be declared
-\tcode{virtual}\iref{class.virtual}
-or pure
-\tcode{virtual}\iref{class.abstract}.
+A prospective destructor can be
+a virtual\iref{class.virtual}
+or pure virtual function\iref{class.abstract}.
```

Comment:
Certainly, a pure virtual function is also virtual, so this seems a bit redundant. But that's a pre-existing defect.

Reply:
_Pure virtual function_ is _technically_ the correct terminology, albeit a tad redundant.

Reply:
I'm arguing on the semantic level. "pure virtual function" implies "virtual function".